.envrc

```
@@ -27,5 +27,13 @@ source_secrets() {
   watch_file "$file"
 }
 
+install_git_hooks() {
+  for path in scripts/git-hooks/*; do
+    filename=$(basename $path)
+    ln -sf ../../scripts/git-hooks/$filename .git/hooks/$filename
```

```suggestion
    ln -sf "../../scripts/git-hooks/$filename" ".git/hooks/$filename"
```

(slightly more defensive if we were ever to use spaces in our paths)